### PR TITLE
Updated README(issue #93)

### PR DESCRIPTION
**Pull request by shikharsoni07**
- I have not changed the icon from the template yet.
- Moved the "Updates to be made each sem" section under "Development Notes"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,228 @@
...
+   python3 gyft.py
+   ```
     - Optional flags: 
     	- `--output` or `o`: Output file (`.ics`) to which the ICS file will be written. **Default:** `timetable.ics`
-    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False`
+    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False` 
 
 4. Enter your roll number, password, security answer and OTP (if required) when prompted
-5. Now you can choose between 2 options, which will be prompted to you:
+5. Your timetable will be saved in the file specified by the `--input` flag (**default:** `data.txt`)
```
Comment (ashwinpra) [inline README.md:127]: Remove this, as we aren't using `data.txt` anymore

File: README.md [hunk 2]
```diff
@@ -1,64 +1,228 @@
...
+<!-- PROJECT SHIELDS -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links-->
+<div align="center">
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
```
Comment (ashwinpra) [inline README.md:12]: This is redirecting to Arpit's LinkedIn page. This could be removed, or just replaced with MetaKGP's LinkedIn

**Review by ashwinpra — CHANGES_REQUESTED** [review]

**Comment by ashwinpra** [comment]
@proffapt @harshkhandeparkar do give your reviews regarding the content of the README

File: README.md (hunk 2 above)
Comment (harshkhandeparkar) [reply README.md:12]: MetaKGP doesn't have a LinkedIn at the moment. So we can link to the wiki maybe.

File: README.md [hunk 3]
```diff
@@ -1,64 +1,228 @@
...
+
+</div>
+
+<!-- PROJECT LOGO -->
+<br />
+<!-- UPDATE -->
+<div align="center">
+  <a href="https://github.com/metakgp/gyft">
+    <img width="140" alt="image" src="https://user-images.githubusercontent.com/86282911/206632284-cb260f57-c612-4ab5-b92b-2172c341ab23.png">
```
Comment (harshkhandeparkar) [inline README.md:21]: I'd suggest commenting out the icon until we have a new one.

File: README.md [hunk 4]
```diff
@@ -1,64 +1,228 @@
...
 
 > **Note** All updates to this repo should reflect, with appropriate refactorisation, in [gyft-serve](https://github.com/metakgp/gyft-serve/)
 
+<p align="right">(<a href="#top">back to top</a>)</p>
 
-## Updates To Be Made Each Sem
+<div id="supports"></div>
 
-The following procedure is to be followed each new semester by the maintainer for updating GYFT to work properly.
+### Supports:
```
Comment (harshkhandeparkar) [inline README.md:77]: This section can be either removed or mention the Python versions it supports and link to the webapp.

File: README.md [hunk 5]
```diff
@@ -1,64 +1,228 @@
...
-Please read CONTRIBUTING.md guide to know more.
+Please read [CONTRIBUTING.md](https://github.com/metakgp/gyft/blob/master/CONTRIBUTING.md) guide to know more.
 
 ## Maintainer(s)
 
 - [Ashwin Prasanth](https://github.com/ashwinpra)
+<!-- CONTACT -->
+<!-- ## Contact -->
+
+<!-- <p> -->
```
Comment (harshkhandeparkar) [inline README.md:174]: This unneeded content can be removed instead of commenting.

File: README.md [hunk 6]
```diff
@@ -1,64 +1,228 @@
...
+[forks-shield]: https://img.shields.io/github/forks/metakgp/gyft.svg?style=flat
+[forks-url]: https://github.com/metakgp/gyft/network/members
+[stars-shield]: https://img.shields.io/github/stars/metakgp/gyft.svg?style=flat
+[stars-url]: https://github.com/metakgp/gyft/stargazers
+[issues-shield]: https://img.shields.io/github/issues/metakgp/gyft.svg?style=flat
+[issues-url]: https://github.com/metakgp/gyft/issues
+[license-shield]: https://img.shields.io/github/license/metakgp/gyft.svg?style=flat
+[license-url]: https://github.com/metakgp/gyft/blob/master/LICENSE.txt
+[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=flat&logo=linkedin&colorB=555
+[linkedin-url]: https://linkedin.com/in/proffapt
```
Comment (proffapt) [inline README.md:228]: Replace these with similar things for wiki and redirecting to wiki.metakgp.org

File: README.md [hunk 7]
```diff
@@ -1,64 +1,228 @@
...
+
+<!-- ## Additional documentation
+
+  - [Changelogs](/.github/CHANGELOG.md)
+  - [License](/LICENSE.txt)
+  - [Security Policy](/.github/SECURITY.md)
+  - [Code of Conduct](/.github/CODE_OF_CONDUCT.md)
+  - [Contribution Guidelines](/.github/CONTRIBUTING.md)
+
+<p align="right">(<a href="#top">back to top</a>)</p> -->
```
Comment (proffapt) [inline README.md:213]: Delete all of this. Not relevant here.

File: README.md [hunk 8]
```diff
@@ -1,64 +1,228 @@
...
+
+
+<!-- USAGE EXAMPLES -->
+<!-- ## Usage -->
+<!-- UPDATE -->
+<!-- <div align="center">
+  <a href="https://github.com/proffapt/PROJECT_NAME">
+    <img width="80%" alt="image" src="https://user-images.githubusercontent.com/86282911/206632640-40dc440e-5ef3-4893-be48-618f2bd85f37.png">
+  </a>
+</div> -->
```
Comment (proffapt) [inline README.md:147]: Add usage examples for deleting the events and any other tags if there are - I lost track of the progress in here

File: README.md [hunk 9]
```diff
@@ -1,64 +1,228 @@
...
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Installation
+
+_Now that the environment has been set up and configured to properly compile and run the project, the next step is to install and configure the project locally on your system._
+<!-- UPDATE -->
+1. Clone the repository
+   ```sh
+   git clone https://github.com/metakgp/gyft
```
Comment (proffapt) [inline README.md:108]:
```suggestion
   git clone --depth 1 https://github.com/metakgp/gyft
```
For the future but won't make any differences now either ¯\_(ツ)_/¯

File: README.md (hunk 4 above)
Comment (proffapt) [reply README.md:77]: Yes, removed all of it, and add the version of python in next section where pre-requisites are mentioned.

File: README.md [hunk 10]
```diff
@@ -1,64 +1,228 @@
...
+<!-- ABOUT THE PROJECT -->
+## About The Project
+<!-- UPDATE -->
+<!-- <div align="center">
+  <a href="https://github.com/proffapt/PROJECT_NAME">
+    <img width="80%" alt="image" src="https://user-images.githubusercontent.com/86282911/206632547-a3b34b47-e7ae-4186-a1e6-ecda7ddb38e6.png">
+  </a>
+</div> -->
+
+_Gets your timetable from ERP and adds it to your Google Calendar or gives you an ICS file which you can add in any common calendar application._
```
Comment (proffapt) [inline README.md:69]:
```suggestion
Gets your timetable from ERP and adds it to your Google Calendar or gives you an ICS file which you can add in any common calendar application.
```
It was not supposed to be italicised.

File: README.md (hunk 3 above)
Comment (proffapt) [reply README.md:21]: agreed

**Review by proffapt — CHANGES_REQUESTED** [review]
image

Why are the shields so small?

File: README.md (hunk 6 above)
Comment (harshkhandeparkar) [reply README.md:228]: Also use `style=for-the-badge`

**Comment by shikharsoni07** [comment]
Updated. Pls review

File: README.md [hunk 11]
```diff
@@ -1,33 +1,112 @@
...
+    <img width="140" alt="image" src="https://user-images.githubusercontent.com/86282911/206632284-cb260f57-c612-4ab5-b92b-2172c341ab23.png">
+  </a> -->
+
+  <h1 align="center">GYFT</h1>
+
+  <p align="center">
+  <!-- UPDATE -->
+    <i>Get Your Freaking Timetable</i>
+    <!-- <br /> -->
+    <!-- <a href="https://github.com/proffapt/PROJECT_NAME"><strong>Explore the docs »</strong></a> -->
```
Comment (harshkhandeparkar) [inline README.md:30]: This comment also seems unnecessary.

File: README.md [hunk 12]
```diff
@@ -1,33 +1,112 @@
...
+
+- [About The Project](#about-the-project)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Development Notes](#development-notes)
+- [Contributing](#contributing)
+- [Maintainer(s)](#maintainers)
+<!-- - [Contact](#contact) -->
```
Comment (harshkhandeparkar) [inline README.md:52]: And these.

File: README.md [hunk 13]
```diff
@@ -1,33 +1,112 @@
...
+<!-- - [Additional documentation](#additional-documentation) -->
+
+</details>
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+<!-- UPDATE -->
+<!-- <div align="center">
+  <a href="https://github.com/proffapt/PROJECT_NAME">
```
Comment (harshkhandeparkar) [inline README.md:63]: Basically remove all unused comments except the icon at the top.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by harshkhandeparkar** [comment]
> Updated. Pls review

Please go through the reviews and "mark as resolved" any changes you have already made.

File: README.md (hunk 9 above)
Comment (shikharsoni07) [reply README.md:75]: should i do this?

**Review by harshkhandeparkar — APPROVED** [review]
LGTM

File: README.md (hunk 9 above)
Comment (ashwinpra) [reply README.md:75]: Yes, you can change it

**Comment by shikharsoni07** [comment]
Done!

**Review by ashwinpra — DISMISSED** [review]
LGTM

File: README.md [hunk 14]
```diff
@@ -1,33 +1,94 @@
...
+   pip install requirements.txt
+   ```
+4. Run the script
+   ```sh
+   python3 gyft.py
+   ```
     - Optional flags: 
     	- `--output` or `o`: Output file (`.ics`) to which the ICS file will be written. **Default:** `timetable.ics`
-    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False`
+    	- `--del-events`: To delete events automatically added by the script before adding new events (explained later) **Default:** `False`
```
Comment (proffapt) [inline README.md:91]: Shift this part to `Usage` section. 
This is only intended to get the things setup for gyft.

Add all the details which aren't there in `Usage` section from here and format it.

Comment (shikharsoni07) [reply README.md:91]: So like all this; points 4 to 6? 
To `usage` section?
image

Comment (proffapt) [reply README.md:91]: yes, this is how you "use" not install/set it up

Comment (proffapt) [reply README.md:91]: Perfect!
And it's fine to commit, there is nothing bad in getting more suggestions.

**Review by proffapt — APPROVED** [review]
LGTM!

**Pull request merged**